next is broken

That's why #1542 fails

https://travis-ci.org/boutproject/BOUT-dev/branches

```
  Linking test_delp2
  Compiling  include/test_derivs.cxx
In file included from ../../externalpackages/googletest/googletest/include/gtest/gtest.h:58:0,
                 from include/test_derivs.cxx:1:
include/test_derivs.cxx: In member function ‘virtual void DerivativesTestAdvection_Sanity_Test::TestBody()’:
include/test_derivs.cxx:302:83: error: ‘IsField3DEqualField3D’ was not declared in this scope
       IsField3DEqualField3D(result, expected, "RGN_NOBNDRY", derivatives_tolerance));
                                                                                   ^
  Compiling  test_gyro.cxx
make[1]: *** [include/test_derivs.o] Error 1
  Compiling  include/bout/test_array.cxx
  Linking test_gyro
```

--- a/tests/unit/include/test_derivs.cxx
+++ b/tests/unit/include/test_derivs.cxx
@@ -299,5 +299,5 @@ TEST_P(DerivativesTestAdvection, Sanity) {
   derivative(velocity, input, result, region);
 
   EXPECT_TRUE(
-      IsField3DEqualField3D(result, expected, "RGN_NOBNDRY", derivatives_tolerance));
+      IsFieldEqual(result, expected, "RGN_NOBNDRY", derivatives_tolerance));
 }

